youtube2zim/scraper.py

```diff
@@ -367,6 +370,15 @@ def validate_dateafter_input(self):
             )
             raise ValueError(f"Invalid dateafter input: {exc}")
 
+    def validate_id(self):
+        self.youtube_id = self.youtube_id.replace(" ", "")
```

Comment:
comment here and group it all:

```py
        # space not allowed in youtube-ID
        self.youtube_id = self.youtube_id.replace(" ", "")
        if self.collection_type == "channel" and len(self.youtube_id) > 24:
            raise ValueError("Invalid ChannelId")
        if ";" in self.youtube_id and self.collection_type == "playlist":
            raise ValueError("Invalid YoutubeId")
```

Reply:
In the second "if" condition I think there should be a "," instead of ";" and "!=" instead of "==" or pls correct me if I am wrong somewhere